modules/javafx.controls/src/test/java/test/javafx/scene/control/ListCellTest.java

```diff
@@ -833,5 +836,79 @@ public void testListCellHeights() {
                 cell.maxHeight(-1), 1);
     }
 
+    @Test
+    public void testChangeIndexWhileEditing_jdk_8264127() {
+        List<EditEvent> events = new ArrayList<EditEvent>();
+        list.setOnEditCancel(e -> {
+            events.add(e);
+        });
+        list.setEditable(true);
+        cell.updateListView(list);
+        cell.updateIndex(1);
+        list.edit(1);
+        cell.updateIndex(0);
+        assertTrue(!cell.isEditing());
```

Comment:
let's not confuse the code reader: `assertTrue(!boolean)` should be `assertFalse(boolean)`